### PR TITLE
[NSH-2024] Sets TBD Event for 2024 and adds two new organizers

### PR DESCRIPTION
**Pull request by GeoffCraig**
This does two things.

1. Sets up a TBD date, we have the drive, but we don't have a date
2. Correctly cancels the 2022 event
3. Adds two new organizers!

### CODE REVIEW

File: static/events/2024-nashville/sponsor-dod.png
Comment (jyee) [inline static/events/2024-nashville/sponsor-dod.png:1]: You probably want to remove this.

File: static/events/2024-nashville/logo.jpg
Comment (jyee) [inline static/events/2024-nashville/logo.jpg:1]: The `logo.jpg` and `logo.png` are dupes of the `logo-square.jpg` and `logo-square.png`. Can you remove the dupes? They're not too big, but every little bit helps fight against repo bloat.

**Review by jyee — CHANGES_REQUESTED** [review]
Minor changes to remove unused images.

File: static/events/2024-nashville/sponsor-dod.png
Comment (GeoffCraig) [reply static/events/2024-nashville/sponsor-dod.png:1]: @jyee for some reason updates to my fork aren't coming into this PR so I'm going to have to close and reopen

**Pull request closed without merging**